docs/source/common/loggers.rst

```diff
@@ -217,6 +217,12 @@ The :class:`~pytorch_lightning.loggers.WandbLogger` is available anywhere except
             # Option 2 for specifically logging images
             self.logger.log_image(key="generated_images", images=[some_img])
 
+To visualize using wandb in a jupyter notebook environment use the following magic line command
```

Comment:
can you extend this example?
visualize after training or during training?

or maybe link this too: https://docs.wandb.ai/guides/track/jupyter#additional-jupyter-features-in-w-and-b